### PR TITLE
add sample apps to ci/cd

### PR DESCRIPTION
**Pull request by bptodorova**
-  Marklogic app can not build. It will be added separately https://github.com/telerik/kendo-react-private/issues/3435

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+          npm run build
+
+      - name: Build React Grid Live Data app
+        working-directory: ./examples/react-grid-live-data
+        run: |
+          npm ci
+          npm run build
+
+      - name: Cleanup
+        run: git clean -xdf
```
Comment (dtopuzov) [inline .github/workflows/ci.yml:120]: No need to clean, machine is disposable.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+    steps:
+      - name: Use NodeJS v18
+        uses: actions/setup-node@v4
+        with:
+          node-version: '18'
+
+      - name: Check out repository
+        uses: actions/checkout@v4
+        with:
+            fetch-depth: 0 # Fetch all branches
```
Comment (dtopuzov) [inline .github/workflows/ci.yml:25]: Why we fetch all branches?

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+          npm i
+          cd ../client    
+          npm i
+          npm run build
+
+      - name: Build React Sales Dashboard app
+        working-directory: ./examples/kendo-react-sales-dashboard
+        run: |
+          npm ci
+          CI= npm run build
```
Comment (dtopuzov) [inline .github/workflows/ci.yml:87]: Are we sure this line is ok, looks unfinished.

File: .github/workflows/cd.yml [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+      - name: Deploy to GH Pages
+        uses: peaceiris/actions-gh-pages@v3
+        with:
+          github_token: ${{ github.token }}
+          publish_dir: ./examples/dist
+          user_name: 'kendo-bot'
+          user_email: 'kendouiteam@progress.com'
+
+      - name: Cleanup
+        run: git clean -xdf
```
Comment (dtopuzov) [inline .github/workflows/cd.yml:43]: No need to clean, the machine is disposable.

File: .github/workflows/ci.yml [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+        working-directory: ./examples/kendo-react-nextjs
+        run: |
+          npm ci
+          npm run build    
+
+      - name: Build React Nodejs Express app
+        working-directory: ./examples/kendo-react-nodejs-express
+        run: |
+          cd server
+          npm i
```
Comment (dtopuzov) [inline .github/workflows/ci.yml:78]: Why not `npm ci`?

**Review by zdravkov — APPROVED** [review]
LG

**Pull request merged**